Remove unneeded local method upgrades_to in 18sj

bentziaxl opened the pull request:
fix #8120:

#8020 updated the tile names. Stockholm gray was changed from 131 to X5.
The local method of upgrades_to?, returns non existent tile 131 when trying to upgrade to gray. The super method handles the upgrade fine, no need to override it.

perwestling reviewed (approved):
I tried it out and it worked fine. Yes, this was unneeded code from the start as the upgrade of Stockholm hex always was legal.

perwestling commented:
If this breaks any game, pin them.

bentziaxl commented:
the tiles had some labels that were changed, added and removed. Maybe some existing label broke the regular upgrade_to? logic and required an override at some point

dolfyner commented:
Hi!
Just wondering if I will get a notice when the issue is fixed? Seeing as my original ticket was closed/merged into this one?

tobymao commented:
issue is fixed but probably not deployed yet

dolfyner commented:
Ok! Will there be a notice of deployment here? Will the deployment break the existing game or can we wait for the deployment and continue it? 
Tx for the help!

tobymao commented:
no notice, if your game breaks, it can be pinned

tobymao commented:
it's deployed

dolfyner commented:
Excellent! Works! Tx again 😁
